utils/xmtpRN/client/client.ts

```diff
@@ -0,0 +1,55 @@
+import { getDbEncryptionKey } from "@utils/keychain/helpers";
+import { TransactionReferenceCodec } from "@xmtp/content-type-transaction-reference";
+import {
+  Client,
+  GroupUpdatedCodec,
+  ReactionCodec,
+  ReadReceiptCodec,
+  RemoteAttachmentCodec,
+  ReplyCodec,
+  StaticAttachmentCodec,
+  TextCodec,
+} from "@xmtp/react-native-sdk";
+import { InstallationId } from "@xmtp/react-native-sdk/build/lib/Client";
+import { config } from "../../../config";
+import { getDbDirectory } from "../../../data/db";
+import { CoinbaseMessagingPaymentCodec } from "../content-types/coinbasePayment";
+import { ConverseXmtpClientType } from "./client.types";
+
+const codecs = [
+  new TextCodec(),
+  new ReactionCodec(),
+  new ReadReceiptCodec(),
+  new GroupUpdatedCodec(),
+  new ReplyCodec(),
+  new RemoteAttachmentCodec(),
+  new StaticAttachmentCodec(),
+  new TransactionReferenceCodec(),
+  new CoinbaseMessagingPaymentCodec(),
+];
+
+export const getXmtpClientFromAddress = async (address: string) => {
```

Comment:
We should name this build .... as it isn't just getting

Reply:
Yes this whole client file need a refactor